app/controllers/api/v0/forem_directories_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V0
+    class ForemDirectoriesController < ApiController
```

Comment:
I was hoping to cache this endpoint at the edge...but since all of this data is based on `Settings` (which uses https://github.com/huacnlee/rails-settings-cached) and not a "Model", there's not a clean way to bust the edge cache anytime these values are changed.

Since these values are cached anyways, it shouldn't be too bad.

Reply:
Should we mark this API as no cache? Because unless you mark it with no cache headers, it will be by network proxies. Don't know how important is to always have fresh results.